src/cloud/components/SeriesCardinalityIncreaseForm.tsx

```diff
@@ -1,41 +1,97 @@
 // Libraries
-import React, {FC} from 'react'
+import React, {FC, useState} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
 
 // Components
 import {
   Form,
-  Button,
   Heading,
+  TextArea,
+  Button,
+  ButtonType,
   ButtonShape,
   InfluxColors,
   ComponentSize,
   ComponentColor,
   HeadingElement,
+  ComponentStatus,
 } from '@influxdata/clockface'
 
+// Actions
+import {showOverlay, dismissOverlay} from 'src/overlays/actions/overlays'
+
+// Selectors
+import {selectQuartzIdentity} from 'src/identity/selectors'
+
 export const SeriesCardinalityIncreaseForm: FC = () => {
+  const dispatch = useDispatch()
+  const quartzIdentity = useSelector(selectQuartzIdentity)
+  const {org: identityOrg} = quartzIdentity.currentIdentity
+
+  const [requestDetails, setRequestDetails] = useState('')
+
+  const handleDetailsValidation = (value: string): string | null => {
+    if (!value) {
+      return 'Request details are required'
+    }
+    return null
+  }
+
+  const isFormValid = (): boolean => {
+    return !!requestDetails
```

Comment:
Nit: I would recommend using `Boolean(requestDetails)` (calling Boolean as a function) over the `!!` operator. Same behavior, but there is no ability to misread `Boolean` during a review.